### ISSUE TITLE
`axisPointer.label.formatter` is not receiving the same parameters as in `RC` release, and also documented

### ISSUE THREAD

**Issue by arashdalir** [apache/incubator-echarts#10868]
### Version
4.2.1

### Steps to reproduce
when defining the `tooltip.axisPointer.label.formatter`, as defined [here](https://echarts.apache.org/en/option.html#tooltip.axisPointer.label.formatter) too, one expects `axis` information to be delivered with `params` (like in screenshot1 - version `4.2.1-rc3`. But this is not the case anymore in `4.2.1` release (as visualised in screenshot2).

screenshot1: 
![image](https://user-images.githubusercontent.com/2896170/61220811-65789b00-a717-11e9-80ab-c8a308faf948.png)

screenshot2:
![image](https://user-images.githubusercontent.com/2896170/61220870-8a6d0e00-a717-11e9-9772-cc9831d0e57c.png)

### What is expected?
to have the same argument as in RC version

### What is actually happening?
well, nothing is being delivered.

### How to Reproduce?
create a chart, define a `tooltip.axisPointer.label.formatter` function and then debug the same js code once with 4.2.1 and once with rc3. you will see the discrepancy right away.

**Comment by Ovilia** [comment]
Please provide a demo based on https://jsfiddle.net/ovilia/n6xc4df3/ .

**Comment by arashdalir** [comment]
okay! found the "issue"!

we have modified the echarts code internally and I wasn't aware of it.

we have modified the code as follows to allow `axis` information to be delivered to formatter function:

```javascript
/**
	 * @param {number} value
	 * @param {module:echarts/coord/Axis} axis
	 * @param {module:echarts/model/Global} ecModel
	 * @param {Object} opt
	 * @param {Array.<Object>} seriesDataIndices
	 * @param {number|string} opt.precision 'auto' or a number
	 * @param {string|Function} opt.formatter label formatter
	 */
	function getValueLabel(value, axis, ecModel, seriesDataIndices, opt)
	{
		value = axis.scale.parse(value);
		var text = axis.scale.getLabel(
			// If `precision` is set, width can be fixed (like '12.00500'), which
			// helps to debounce when when moving label.
			value, {precision: opt.precision}
		);
		var formatter = opt.formatter;

		if(formatter)
		{
			var params = {
				value: getAxisRawValue(axis, value),
				axis: axis, // <- this is added to code
				seriesData: []
			};
			each$1(seriesDataIndices, function (idxItem){
				var series = ecModel.getSeriesByIndex(idxItem.seriesIndex);
				var dataIndex = idxItem.dataIndexInside;
				var dataParams = series && series.getDataParams(dataIndex);
				dataParams && params.seriesData.push(dataParams);
			});

			if(isString(formatter))
			{
				text = formatter.replace('{value}', text);
			}
			else
			{
				if(isFunction$1(formatter))
				{
					text = formatter(params);
				}
			}
		}

		return text;
	}
```
